edx_toggles/tests/test_waffle.py

```diff
@@ -0,0 +1,37 @@
+"""
+Unit tests for waffle classes.
+"""
+
+from django.test import TestCase
+
+# TODO import from edx_toggles.toggles once we remove the legacy classes from the exposed API
+from edx_toggles.toggles.internal.waffle.flag import WaffleFlag
+from edx_toggles.toggles.internal.waffle.switch import WaffleSwitch
+
+
+class WaffleFlagTests(TestCase):
```

Comment:
Ultimately, is this the only planned coverage? Do we need a TODO to create or move tests?

Reply:
I didn't see fit to introduce more tests, as most of them are already present in `test_legacy_waffle.py`. I guess the tests from test_legacy_waffle should be moved here once the legacy classes are removed.

Reply:
Can you add this as a task, or subtask, so that we don't drop coverage assuming the tests already exist?  It could also be a TODO  comment in the docstring of this file.

Reply:
I added a comment in the docstring of test_legacy_waffle.py.